docs/source/noise/row_noise.rst

```diff
@@ -89,3 +89,50 @@ parameter:
 
 When applying :code:`omit_row` noise, each row of data is selected for omission
 independently with probability :code:`row_probability`.
+
+Duplicate with guardian
+-----------------------
+
+A known entity resolution challenge is children being reported multiple
+times at different addresses. This can occur when family structures are
+complex and children might spend time at multiple households. A related
+challenge occurs with college students, who often are counted both at their
+university and at their guardian’s home address.
+
+To facilitate this type of error, we have simulants assigned to guardians
+within the simulation. Sometimes, those guardians may move and live at
+different addresses than their dependents. In this case, there is an
+opportunity for duplication. Since this mechanism occurs within the
+simulation, there is a natural maximum that we will impose in the
+noise function.
+
+Guardian-based duplication is applied to two mutually exclusive categories of
+simulants based on age and GQ status: Simulants younger than 18  (<18) and not
+in GQ and simulants under 24 (<24) and in college GQ.
+
+For each of the two categories of simulants, the maximum duplication rate will
+be calculated based on those who have a guardian living at a different address
+in the sim. Note that all simulants in *college* GQ are initialized with a
+guardian living at a different address, but this is not true for simulants in
+other types of GQ, so both maximum duplication rates will be less than
+100%. If you as the user select a duplication rate that is higher than the 
+calculated maximum rate in the sim, a warning will be issued explaining that 
+the selected rate is greater than the maximum available, and the actual rate of 
+duplication should be set to the maximum for the specified simulant category.
+
+This noise type is called :code:`duplicate_with_guardian` in the configuration. 
+It takes two parameters:
+
+.. list-table:: Parameters to the duplicate_with_guardian noise type
+  :widths: 1 5 3
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`row_probability_in_households_under_18`
+    - The probability that a simulant under 18 in a household is recorded twice.
+    - 0.05 (5%)
```

Comment:
Is the default really 5% for both? It seems like it should be much lower for this category than the college category.

Reply:
Hmm yeah I put 5% based on [this slack thread](https://ihme.slack.com/archives/C02KUQ9LX32/p1703196328009399?thread_ts=1703183820.206599&cid=C02KUQ9LX32) from a couple weeks ago, assuming this is what Jim implemented. I might have misunderstood something though

Reply:
You're right, that [is what Jim implemented](https://github.com/ihmeuw/pseudopeople/pull/361/files#diff-6cd04081d627fca6439a9f697ad36a036f010132f10ae66387d381018801b9adR28-R29). @NathanielBlairStahn can you shed some light?

Reply:
I'm pretty sure 5% was just a made-up number, and we used the same percentage for both groups for simplicity (5% for both groups was written in the docs before I edited them, and I just left it). @zmbc do you have any more realistic estimates?

Reply:
I don't have any data on this, no. Seems like the most relevant thing the Census Bureau has recently published is [this blog](https://www.census.gov/newsroom/blogs/random-samplings/2021/04/how_we_unduplicated.html) but it doesn't include even ballpark numbers.

I expect we will want to adjust our `row_probability_in_households_under_18`; I'd assume that 5% would cause a warning.

Reply:
Let's go with 2% for under 18, based on [Zeb's slack post](https://ihme.slack.com/archives/C02KUQ9LX32/p1705083307080909), and leave college students at 5% for now, until we can derive a better estimate from the sources Krista sent.

Reply:
```suggestion
    - 0.02 (2%)
```